<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulation U-Tracking</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget { background-color: #001d26; }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>280</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {  color: white; 
background-color: rgba(255, 170, 0,150);
border-width: 0px;  
border-radius: 10px;
 } 
QPushButton:hover {
	background-color: rgb(0, 100, 0,255)

}
QPushButton:pressed {
background-color: rgb(0, 0, 255,250);

}

QPushButton:disabled{
background-color: rgba(0, 0, 0,40);
}
</string>
    </property>
    <property name="text">
     <string>TAKEOFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_land">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>380</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {  color: white; 
background-color: rgba(255, 0, 0,150);
border-width: 0px;  
border-radius: 10px;
 } 
QPushButton:hover {
	background-color: rgb(0, 100, 0,255)

}
QPushButton:pressed {
background-color: rgb(0, 0, 255,250);

}

QPushButton:disabled{
background-color: rgba(0, 0, 0,40);
}
</string>
    </property>
    <property name="text">
     <string>LAND</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_arm">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {  color: white; 
background-color: rgba(0, 170, 0,150);
border-width: 0px;  
border-radius: 10px;
 } 
QPushButton:hover {
	background-color: rgb(0, 100, 0,255)

}
QPushButton:pressed {
background-color: rgb(0, 0, 255,250);

}

QPushButton:disabled{
background-color: rgba(0, 0, 0,40);
}
</string>
    </property>
    <property name="text">
     <string>ARM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_WP1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>380</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {  color: white; 
background-color: rgba(255, 170, 0,150);
border-width: 0px;  
border-radius: 10px;
 } 
QPushButton:hover {
	background-color: rgb(0, 100, 0,255)

}
QPushButton:pressed {
background-color: rgb(0, 0, 255,250);

}

QPushButton:disabled{
background-color: rgba(0, 0, 0,40);
}
</string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>34</y>
      <width>601</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_al">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 255, 0);</string>
       </property>
       <property name="text">
        <string>N/A</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Lat::</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLabel" name="label_15">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Heading(deg)::</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="Line" name="line_12">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="Line" name="line_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_lat">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 255, 0);</string>
       </property>
       <property name="text">
        <string>N/A</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_13">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Altitude(m)::</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label_lon">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 255, 0);</string>
       </property>
       <property name="text">
        <string>N/A</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="Line" name="line_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QLabel" name="label_hd">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 255, 0);</string>
       </property>
       <property name="text">
        <string>N/A</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Lon::</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="6">
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="6">
      <widget class="Line" name="line_13">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>611</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgba(0,0, 0,150);
border-radius: 20px; </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="picture.qrc">:/Griffin.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mode">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>290</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {  color: white; 
background-color: rgba(0, 170, 0,150);
border-width: 0px;  
border-radius: 10px;
 } 
QPushButton:hover {
	background-color: rgb(0, 100, 0,255)

}
QPushButton:pressed {
background-color: rgb(0, 0, 255,250);

}

QPushButton:disabled{
background-color: rgba(0, 0, 0,40);
}
</string>
    </property>
    <property name="text">
     <string>GUILD</string>
    </property>
   </widget>
   <zorder>label_8</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_land</zorder>
   <zorder>pushButton_arm</zorder>
   <zorder>pushButton_WP1</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>label</zorder>
   <zorder>pushButton_mode</zorder>
  </widget>
 </widget>
 <resources>
  <include location="picture.qrc"/>
 </resources>
 <connections/>
</ui>
